extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -2159,6 +2159,63 @@ describe('amp-a4a', () => {
     });
   });
 
+  describe('refresh', () => {
+    let sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should effectively reset the slot and invoke given callback', () => {
```

Comment:
Working on bullet points 1, 2, and 4 (this will need to be part of doubleclick impl's tests). Not sure what you mean for the third one, especially by resume. Can you elaborate?